src/utils/delegations/signStakingTx.ts

```diff
@@ -1,16 +1,18 @@
+import * as ecc from "@bitcoin-js/tiny-secp256k1-asmjs";
 import { Transaction, networks } from "bitcoinjs-lib";
+import * as bitcoin from "bitcoinjs-lib";
 import { stakingTransaction } from "btc-staking-ts";
 
+import { signPsbtTransaction } from "@/app/common/utils/psbt";
 import { GlobalParamsVersion } from "@/app/types/globalParams";
-import { apiDataToStakingScripts } from "@/app/utils/apiDataToStakingScripts";
-import { signPsbtTransaction } from "@/app/utils/psbt";
-import { isTaproot } from "@/app/utils/wallet";
-import { UTXO, WalletProvider } from "@/app/utils/wallet/wallet_provider";
+import { apiDataToStakingScripts } from "@/utils/apiDataToStakingScripts";
+import { isTaproot } from "@/utils/wallet";
+import { UTXO, WalletProvider } from "@/utils/wallet/wallet_provider";
 
 import { getStakingTerm } from "../getStakingTerm";
 
 import { txFeeSafetyCheck } from "./fee";
-
+bitcoin.initEccLib(ecc);
```

Comment:
might be useful to create a singleton instead of initing this everywhere it's used